.github/workflows/android.yml

```diff
@@ -91,22 +106,24 @@ jobs:
                 -D CMAKE_CXX_COMPILER_LAUNCHER=ccache \
                 -D SYSTEM_QT=ON \
                 -D ANDROID_SDK=/usr/local/lib/android/sdk/ \
+                -D ANDROID_SDK_ROOT=/usr/local/lib/android/sdk/ \
                 -D ANDROID_NDK_VERSION="${ANDROID_NDK_VERSION}" \
                 -D ANDROID_BUILD_TOOLS_VERSION="${ANDROID_BUILD_TOOLS_VERSION}" \
                 -D VCPKG_TARGET_TRIPLET="${{ matrix.triplet }}" \
                 -D WITH_ALL_FILES_ACCESS="${{ matrix.all_files_access }}" \
                 -D WITH_SPIX=OFF \
+                -D BUILD_WITH_QT6=ON \
+                -D CMAKE_PREFIX_PATH=${Qt6_DIR} \
```

Comment:
If possible we should replace that with something like `Qt6_ROOT=${Qt6_DIR}`, to be tested ...